package.json

```diff
@@ -28,7 +28,7 @@
 	"activationEvents": [
 		"onCommand:quarkusTools.createMavenProject",
 		"onCommand:quarkusTools.addExtension",
-		"*"
+		"onLanguage:properties"
```

Comment:
No that's too broad. Just `application.properties` files for now.

We might need to create a Quarkus application properties type like in https://github.com/spring-projects/sts4/blob/0714b6a69309789ac85441062698153b0ac4d9c6/vscode-extensions/vscode-spring-boot/package.json#L55-L66

Reply:
quarkus application properties type might be explored in another PR. Will be interesting to see how it will conflicts with vscode-spring-boot, since they'll likely use the same triggers.